plugins/wysiwygarea/plugin.js

```diff
@@ -716,3 +796,14 @@ CKEDITOR.config.disableNativeSpellChecker = true;
  * @param {CKEDITOR.editor} editor This editor instance.
  * @param {CKEDITOR.dom.element} data The element being added.
  */
+
+/**
+ * Editor detachable parent. Native DOM object.
+ *
+ * Editor `wysiwygarea` iframe will be recreated whenever this element will be reattached to DOM.
```

Comment:
I change the name to `observableParent`. Because we can't observe element which is detachable. If so, then no changes will be detected. We have to choose at least the parent of the detachable element.